### PR TITLE
[Model] Add stable diffusion model based on fastdeploy

### PR DESCRIPTION
**Pull request by joey12300**
### PR types

New features
### PR changes

Others
### Describe

Add stable diffusion model based on fastdeploy

### CODE REVIEW

File: examples/pipelines/stable_diffusion/export_model.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
+# Copyright (c) 2022 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
```
Comment (ZeyuChen) [inline examples/pipelines/stable_diffusion/export_model.py]: 后面注意这里的license不能是这个。
目录改为
examples/multimodal/stable_diffusion

File: examples/multimodal/stable_diffusion/README.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+[PPDiffusers](https://github.com/PaddlePaddle/PaddleNLP/tree/develop/ppdiffusers)是一款支持跨模态（如图像与语音）训练和推理的扩散模型（Diffusion Model）工具箱，其借鉴了🤗 Huggingface团队的[Diffusers](https://github.com/huggingface/diffusers)的优秀设计，并且依托[PaddlePaddle](https://github.com/PaddlePaddle/Paddle)框架和[PaddleNLP](https://github.com/PaddlePaddle/PaddleNLP)自然语言处理库。下面介绍如何使用FastDeploy将PPDiffusers提供的Diffusion模型进行高性能部署。
+
+### 依赖安装
+
+模型导出需要依赖`paddlepaddle`, `paddlenlp`以及`ppdiffusers`，可使用`pip`执行下面的命令进行快速安装。
+
+```shell
+pip install -r requirements_paddle.txt
+```
+
```
Comment (jiangjiajun) [inline examples/multimodal/stable_diffusion/README.md]: 如刚才沟通
- 提供预训练模型
- 拆分成独立的export.md

**Review by jiangjiajun — CHANGES_REQUESTED** [review]

File: examples/multimodal/stable_diffusion/README.md (hunk 2 above)
Comment (joey12300) [reply examples/multimodal/stable_diffusion/README.md]: Done

File: examples/multimodal/stable_diffusion/README.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+|----------|--------------|
+| [CompVis/stable-diffusion-v1-4](https://bj.bcebos.com/fastdeploy/models/stable-diffusion/CompVis/stable-diffusion-v1-4.tgz) | PNDM |
+| [runwayml/stable-diffusion-v1-5](https://bj.bcebos.com/fastdeploy/models/stable-diffusion/runwayml/stable-diffusion-v1-5.tgz) | EulerAncestral |
+
+## 环境依赖
+
+在示例中使用了PaddleNLP的CLIP模型的分词器，所以需要执行以下命令安装依赖。
+
+```shell
+pip install paddlenlp paddlepaddle-gpu
```
Comment (jiangjiajun) [inline examples/multimodal/stable_diffusion/README.md:22]: 后期将FastTokenizer接口也封装一份在FastDeploy，避免引入过多依赖

**Review by jiangjiajun — APPROVED** [review]

**Pull request merged**